clients/src/main/java/org/apache/kafka/clients/admin/ExpireDelegationTokenOptions.java

```diff
@@ -25,10 +25,19 @@
  * The API of this class is evolving, see {@link Admin} for details.
  */
 @InterfaceStability.Evolving
-public class ExpireDelegationTokenOptions extends AbstractOptions<ExpireDelegationTokenOptions> {
+public class ExpireDelegationTokenOptions
+    extends AbstractOptions<ExpireDelegationTokenOptions> {
+
     private long expiryTimePeriodMs = -1L;
 
-    public ExpireDelegationTokenOptions expiryTimePeriodMs(long expiryTimePeriodMs) {
+    /**
+     * @param expiryTimePeriodMs the time period until we should expire this token.
+     * {@code expiryTimePeriodMs} >= 0: the token will update the `expiration timestamp` if the current expiration timestamp is small than (now + expiryTimePeriodMs).
```

Comment:
Sorry that could you please revise it to "{@code expiryTimePeriodMs} >= 0: the token will update the `expiration timestamp` to min(now + expiryTimePeriodMs, maxTimestamp)`

Reply:
done.